iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -556,15 +578,18 @@ public void testCTASFailureRollback() throws IOException {
     testTables.createTable(shell, "source", HiveIcebergStorageHandlerTestUtils.CUSTOMER_SCHEMA,
         fileFormat, HiveIcebergStorageHandlerTestUtils.CUSTOMER_RECORDS);
 
-    try {
-      shell.executeStatement(String.format("CREATE TABLE target STORED BY '%s' AS SELECT * FROM source",
-          HiveIcebergStorageHandler.class.getName()));
-    } catch (Exception e) {
-      // expected error
-    }
+    String[] partitioningSchemes = {"" /* unpartitioned */, "PARTITIONED BY (dept)", "PARTITIONED BY (dept, name)"};
+    for (String partitioning : partitioningSchemes) {
+      try {
+        shell.executeStatement(String.format("CREATE TABLE target %s STORED BY '%s' AS SELECT * FROM source",
```

Comment:
nit: after @lcspinter's patch we can use STORED BY ICEBERG

Reply:
Done